mkdocs.yml

```diff
@@ -54,7 +54,7 @@ nav:
               - Introduction: v3/features/helpers/intro.md
               - UrlHelper: v3/features/helpers/url-helper.md
               - ServerUrlHelper: v3/features/helpers/server-url-helper.md
-              - 'Body Parsing Middleware': v3/features/helpers/body-parse.md
+              - 'Body Params Middleware': v3/features/helpers/body-params.md
```

Comment:
For this change, a redirect is needed. Add the following at the end of the file:

```yaml
plugins:
  - search
  - redirects:
        redirect_maps:
            v3/features/helpers/body-parse.md: v3/features/helpers/body-params.md
```
Compare with the MkDocs configuration of laminas-form: https://github.com/laminas/laminas-form/blob/c6a69c93e9c445b9a2e08cd7a4746bc724bd4ffe/mkdocs.yml#L192-L196